### PR TITLE
ready for pR

### PR DESCRIPTION
**Pull request by cindyciclta**
Made my test work, so I added all my tests in this PR.

### CODE REVIEW

File: src/service.cc [hunk 1]
```diff
@@ -369,10 +394,52 @@ void ServiceLayer::terminate_monitor(chirp::User &user,
         users_monitoring); // save back all users still monitoring
   }
 }
-std::vector<chirp::Chirp> ServiceLayer::stream(const std::string hashtag) {
+std::vector<chirp::Chirp>
+ServiceLayer::stream(const std::string hashtag, std::time_t seconds,
```
Comment (barath) [inline src/service.cc:398]: Minor: const reference.

**Review by barath — APPROVED** [review]

File: src/service.h [hunk 2]
```diff
@@ -51,10 +51,17 @@ class ServiceLayer final {
...
-  // hashtag. If so, save that chirp into a proto Hashtags.
+  std::vector<chirp::Chirp> stream(const std::string hashtag,
+                                   std::time_t seconds,
+                                   int64_t microseconds_since_epoch);
+  // This function is called in chirp() to determine if the text consist of
+  // an hashtag. If so, save that chirp into a proto Hashtags.
   std::string CheckIfHaveHashtag(const std::string &text);
+  // This function is a helper function to find multiple hashtags in a chirp, if
+  // exists
+  std::vector<std::string> CheckIfHaveMultipleHashtags(const std::string &text);
```
Comment (apetranik) [inline src/service.h:62]: Should these be private helper functions?

**Review by apetranik — APPROVED** [review]
Looks good

File: src/service.h (hunk 2 above)
Comment (cindyciclta) [reply src/service.h:62]: I wanted to test the function. I thought it's okay to put them here since service.h is like a middle layer class?

**Pull request merged**